frontend/src/app/shared/sidepanel/sidepanel.component.html

```diff
@@ -3,6 +3,7 @@
     (backdropClick)="close()"
     [cdkConnectedOverlayHasBackdrop]="true"
     [cdkConnectedOverlayHeight]="'100vh'"
+    [cdkConnectedOverlayWidth]="'800px'"
```

Comment:
?

Reply:
If you resolve a conversation, please provide some context as to the answer if it is not obvious. What was the question? Why is this okay, not okay, not relevant, ....

Reply:
the question was why is this needed now and wasn't before?
Because angular/material changed the Sidepanel it now has 100vw/100% instead of the 800px it used to have